### ISSUE TITLE
Confirm payment in mempool on receive screen

### ISSUE THREAD

**Issue by m2049r** [m2049r/xmrwallet#68]
The QR receive screen should show when payments have reached the requested amount with the specified payment id.

This probably also means "freezing" the receive screen so the amount and payment id cannot be changed as it is now.

Further down the line we may have a screen only for awaited transactions ...

**Comment by golere** [comment]
good idea. after implementing this function, monerujo could be used as a stand-alone point-of-sale app. Similar to [Kasisto](https://github.com/amiuhle/kasisto) (which is not stand-alone).
